changes.d/init/permanent-delete.md

```
@@ -0,0 +1,3 @@
+ -  Added permanent delete warning for Linux or other UNIX-like system users
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)